woof-code/rootfs-skeleton/usr/sbin/pcur

```diff
@@ -19,14 +19,6 @@ func_switch (){
 		if [ `id -u` -eq 0 ] ; then
 			rm -f /home/spot/.icons/default
 		fi
-	elif [[ -d /usr/share/icons/"$1" ]]; then
-		mkdir -p $HOME/.icons/
-		ln -snf /usr/share/icons/"$1" $HOME/.icons/default
-		if [ `id -u` -eq 0 ] ; then
-			mkdir -p /home/spot/.icons/
-			ln -snf /usr/share/icons/"$1" /home/spot/.icons/default
```

Comment:
Without the symlinks under /home/spot, X applications running as spot use the default cursor

Reply:
as above, cursor in ptheme is set by pcur on cli